code/game/mecha/mecha.dm

```diff
@@ -326,19 +332,33 @@
 		step_in = initial(step_in)
 	var/move_result = 0
 	var/move_type = 0
+	var/step_malus = 0
```

Comment:
maybe add a comment detailing what this local var is for? Not sure what it's purpose is just looking at the name